src/redux/reducer.js

```diff
@@ -238,6 +262,35 @@ const initialState = {
     ...paramsInitialState,
     limitReductionModified: false,
     lastCompletedComputation: null,
+    // Results filters
+    loadflowResultFilter: {
```

Comment:
all these keys are declared as contants in each result type utils, use them

Reply:
Done